src/extension/background.js

```diff
@@ -1,14 +1,13 @@
 import { makeLog } from "../lib/utils";
 import { normalizeTheme, colorToCSS } from "../lib/themes";
+import { bgImages } from "../lib/themes";
```

Comment:
The `bgImages` declaration can go in the line above since `normalizeTheme`, `colorToCSS`, and `bgImages` are all imported from the same file (`src/lib/themes.js`).

So it should be: 
```js
import { normalizeTheme, colorToCSS, bgImages } from "../lib/themes";
```

This applies to the other occurrences / changes you made of `bgImages` as well.

Reply:
@cedricium Thanks for reviewing.